packages/bootleg-app-contracts/truffle-config.js

```diff
@@ -1,5 +1 @@
-if (__dirname.includes('coverageEnv')) {
-  module.exports = require('../../../truffle-config');
-} else {
-  module.exports = require('../../truffle-config');
-}
+module.exports = require('bootleg-common/truffle-config')(__dirname);
```

Comment:
solidity-coverage copied all the code into a `coverageEnv` folder. sol-coverage doesn't 🎉